src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/WaitAzureRmDataLakeAnalyticsJob.cs

```diff
@@ -75,7 +70,7 @@ public override void ExecuteCmdlet()
                 WriteVerboseWithTimestamp(string.Format(Resources.WaitJobState, jobInfo.State));
                 Thread.Sleep(WaitIntervalInSeconds*1000);
```

Comment:
Fixed, and I found other places where I was calling Thread.Sleep directly and replaced it with this.